Shoot/README.md

```diff
@@ -15,4 +15,78 @@ $ cordova prepare ios
 $ open platforms/ios/HelloCordova.xcodeproj/
 ```
 
-Then follow the directions printed to tweak the native projects to be able to use the oauth plugin. See details instruction in [aerogear-oauth2-cordova](https://github.com/aerogear/aerogear-oauth2-cordova/blob/master/README.md#workaround-for-ios).
\ No newline at end of file
+Then follow the directions printed to tweak the native projects to be able to use the oauth plugin. See details instruction in [aerogear-oauth2-cordova](https://github.com/aerogear/aerogear-oauth2-cordova/blob/master/README.md#workaround-for-ios).
+
+## Facebook setup 
+
+### Step1: Setup facebook to be a facebook developer:
+
+- Go to [Facebook dev console](https://developers.facebook.com/products/login/)
+- Click Apps->Register as a Developer
+- enter password
+- accept policy
+- send confirmation code to SMS
+- once received enter code
+
+### Step2: Create a new app on facebook console
+
+- Click apps-> Create a new app
+- add display name: Shoot
+- deal with difficult catcha
```

Comment:
I need to choice the platform

![screenshot 2015-02-11 15 31 32](https://cloud.githubusercontent.com/assets/13337/6152362/a7aa1ca0-b203-11e4-9872-22db645f64e7.png)